site/src/_pages/docs/3_embedding-blocks.mdx

```diff
@@ -10,28 +10,62 @@ You can also [check out the source code](https://github.com/hashintel/hash/tree/
 You can browse prototype blocks in the [Block Hub](https://blockprotocol.org/hub) right now.
 You can view their schema (the structure of data they accept), and see how they look given different data.
 
-You can also [generate an API key](https://blockprotocol.org/settings/api-keys) to search our API for available blocks:
+### Block API
 
-`https://blockprotocol.org/api/blocks?q=text_query`
+You can also [generate an API key](https://blockprotocol.org/settings/api-keys) to search our API for available blocks.
 
-To authenticate with the API, pass your API key in an `x-api-key` header.
+The API endpoint is `https://blockprotocol.org/api/blocks`.
 
-It is also possible to search blocks by data structure: allowing you to provide a data object, and be told which blocks can display or edit it.
+You must authenticate with the API by passing your API key in an `x-api-key` header.
+
+For example, to retrieve a list of all blocks:
+
+```javascript
+fetch("https://blockprotocol.org/api/blocks", {
+  headers: {
+    "content-type": "application/json",
+    "x-api-key": "YOUR_API_KEY",
+  },
+})
+  .then((resp) => resp.json())
+  .then(console.log);
+```
+
+The following query parameters are available. All are optional, and may be combined to narrow results by a combination of filters:
+
+- `author`: search for blocks with a matching author.
+- `license`: search for blocks with a matching license.
+- `name`: search for blocks with a matching name.
+- `q`: search for blocks with a matching name or author.
```

Comment:
Why `q`?

Reply:
Just a general query?

Reply:
Just a general query - we can rename this. 

When we implement a block searching thing it might indicate better parameters / ways of combining filters.